src/components/MagPage/styles/magazinePage.css

```diff
@@ -0,0 +1,31 @@
+.bkgImage {
+  min-height: 100%;
+  min-width: 1024px;
+  width: 100%;
+  height: auto;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: -1;
+
+  -webkit-filter: blur(5px);
```

Comment:
Should all of the other CSS vendor prefixes be included as well? How does this look/work across the other browsers?